### ISSUE TITLE
List of Color Codes for Quests.yml ?

### ISSUE THREAD

**Issue by l1amm** [CitizensDev/Citizens#668]
Title says it, Does anyone know more color codes for Quests.yml?
I know <g> for green and <y> for yellow and br for new line and thats about it. How do you do the rest of the text colors? anyone have a list?
Thanks!

**Comment by Thyadar** [comment]
Would like to know that too. I think there has to be a possibility to set it back to white for example.

**Comment by bitfreeze** [comment]
Well I went through the source code and found these:

Color codes:
<b>&lt;black&gt;</b> or <b>&0</b> - Black
<b>&lt;dblue&gt;</b> or <b>&1</b> - Dark Blue
<b>&lt;dgreen&gt;</b> or <b>&2</b> - Dark Green
<b>&lt;daqua&gt;</b> or <b>&3</b> - Dark Aqua/Cyan
<b>&lt;dred&gt;</b> or <b>&4</b> - Dark Red
<b>&lt;dpurple&gt;</b> or <b>&5</b> - Dark Purple
<b>&lt;gold&gt;</b> or <b>&6</b> - Gold/Orange
<b>&lt;gray&gt;</b> or <b>&7</b> - Gray
<b>&lt;dgray&gt;</b> or <b>&8</b> - Dark Gray
<b>&lt;blue&gt;</b> or <b>&9</b> - Blue
<b>&a</b> or <b>&lt;g&gt;</b> - Green
<b>&b</b> - Aqua/Cyan
<b>&c</b> - Red
<b>&d</b> - Purple
<b>&e</b> or <b>&lt;y&gt;</b> - Yellow
<b>&f</b> - White

Extra formatting & variables:
<b>&lt;br&gt;</b> - text line break
<b>&lt;h&gt;</b> - player health
<b>&lt;name&gt;</b> - player name
<b>&lt;world&gt;</b> - player's current world
<b>&lt;npc&gt;</b> - NPC name
<b>&lt;npcid&gt;</b> - NPC ID
<b>&lt;delay&gt;</b> - delays the text for 1 server tick
<b>&lt;delayN&gt;</b> - delays the text for N server ticks

P.S.:
- as you can see above, <b>&lt;g&gt;</b> and <b>&lt;y&gt;</b> are actually hard-coded.  Only those two colors.
- the colors green, aqua, red, lpurple, yellow and white would be equivalent to &a to &f, but due to a coding error they probably won't work.
- if you're planning to start your message with a "&" color code, either enclose the whole message between single quotation marks ( ' ), or add a leading < > color code, i.e.:  &lt;g&gt;&dThis is a purple message.

Enjoy, and happy questing!  ;-)

**Comment by l1amm** [comment]
Yea this issue was posted a long time ago before the wiki page had this on it. I already saw this on wiki page but thanks!

**Comment by bitfreeze** [comment]
You're welcome... I only thought that a month old issue - still open and without a reply - could have not been solved yet, hence my answer. Also, I got to this page while googling, trying to find about the colors to configure my server, and failing on that search. (facepalm)

However there are odds that come for good. I had found the wiki page for it, I would not have discovered about the &lt;delay&gt; tag, nor noticed that we can use the & color codes (which in my opinion are much easier) and that the &lt;green&gt; ... &lt;white&gt; ones are bugged.  Also, if I'm not mistaken, &lt;[color number]&gt; for those same colors won't work due to that bug.  Although when I tried using &lt;9&gt; it didn't work either... no clue why.
